edit/namespace/ResourceQuota.vue

```diff
@@ -36,7 +36,11 @@ export default {
   },
   data() {
     return {
-      limitsCPU:      null, limitsMem:     null, reqCPU:        null, reqMem:        null, originalQuota: {}
```

Comment:
Its not currently being used but we will be adding resource quotas in the future and this could be updated to support additional targets like we do in Ember UI.